test/e2e/clients.go

```diff
@@ -29,10 +29,10 @@ func init() {
 		klog.Exit(err.Error())
 	}
 
-	// in case we need to work with OLM types
-	if err := operatorsv1alpha1.AddToScheme(scheme.Scheme); err != nil {
-		klog.Exit(err.Error())
-	}
+	//// in case we need to work with OLM types
```

Comment:
I didn't see this removed later. maybe remove if not needed.

Reply:
done, thx (also the one above this)